engine/hatchery/marathon/types.go

```diff
@@ -13,10 +13,10 @@ type HatcheryConfiguration struct {
 	hatchery.CommonConfiguration
 
 	// MarathonURL "marathon-api"
-	MarathonURL string `default:"" commented:"true" comment:"URL of your marathon"`
+	MarathonURL string `toml:"url" default:"https://lb.gra-1.containers.ovh.net/marathon/yourstack/" commented:"true" comment:"URL of your marathon"`
```

Comment:
this doesn't look like a marathon cluster address. Prefer something like this: http://10.241.1.71:8080,10.241.1.72:8080,10.241.1.73:8080